docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '2'
+services:
+  web:
+    image: openstreetmap-website:v1
+    build:
+      context: ..
```

Comment:
Why not put the Dockerfiles in this directory?

Reply:
Which directory? Right now all docker related file are in one directory named `docker`.

Reply:
Oh wait, nevermind.